src/main/java/digital/slovensko/autogram/drivers/NativePkcs11SignatureToken.java

```diff
@@ -57,22 +59,28 @@ private void runContextSpecificLoginIfNeeded(Signature signature, PrivateKey pk)
             // TODO cache & short-circuit
             var p11 = getP11(signature);
             var sessionId = getSessionId(signature);
-            var keyID = getKeyID(pk);
```

Comment:
Toto som strcil do isAlwaysAuthenticate aby to zbytocne nevolalo nejaku pomalu kartovu operaciu ak tam je ten force parameter true (co casto bude).